maintenance_curation.pt.Rmd

```diff
@@ -6,213 +6,215 @@ aliases:
 
- Os pacotes "Incubadora" são pacotes em desenvolvimento criados pela equipe ou por
+Os pacotes de "incubadora" são pacotes em desenvolvimento criados pela equipe ou por
 membros da comunidade como parte de projetos comunitários, como os criados por
-em não conferências
+em desconferências.
 
-- Os pacotes da incubadora ficarão na organização "ropenscilabs".
+- Os pacotes de incubadora ficarão na organização "ropenscilabs" no GitHub.
 
-- Os pacotes da incubadora aparecerão no registro de pacotes com o símbolo
-  "tag "incubadora
+- Os pacotes de incubadora aparecerão no registro de pacotes com a
+  tag "incubator".
 
-- Os pacotes da incubadora não serão exibidos no site por padrão, mas você pode usar a tag "Incubator" para fazer isso.
-  as páginas de pacotes incluirão uma guia "pacotes experimentais".
+- Os pacotes de incubadora não serão exibidos no site por padrão, mas
+  as páginas de pacotes incluem uma guia especial de "pacotes experimentais".
 
-- Os pacotes da incubadora e seus documentos serão criados pelo rOpenSci
-  [sistema](https://status.ropensci.org/). Esse sistema não envia notificações
-  mas gera resultados como status de commit do GitHub (marca de seleção vermelha ou cruz vermelha).
+- Os pacotes da incubadora e seus documentos serão criados pelo
+  [sistema](https://status.ropensci.org/) da rOpenSci. Esse sistema não envia notificações
+  mas gera resultados como o status de commit do GitHub (o *red check mark* ou o *red cross*).
   Os documentos indicarão claramente que o pacote é experimental.
 
-- Semestralmente ou anualmente, a rOpenSci entrará em contato com os mantenedores da incubadora
-  sobre repositórios com pelo menos três meses de idade, perguntando sobre
-  sobre o status de desenvolvimento e as preferências dos autores para migração para o
-  revisão por pares, ropensci-archive ou para organizações de autores. Baseado em
-  nas respostas, o pacote será migrado imediatamente, a revisão por pares
+- Semestralmente ou anualmente, a rOpenSci entrará em contato com os mantenedores desses pacotes de incubadora
+  sobre repositórios que tenham pelo menos três meses de idade, perguntando
+  sobre o status de desenvolvimento e as preferências dos autores sobre uma migração para o
+  processo de revisão por pares, ou para o "ropensci-archive", ou para uma organização dos autores. Baseado em
+  nas respostas, o pacote será migrado imediatamente, e a revisão por pares
   será iniciada, ou a migração será adiada para a próxima
-  revisão. Os pacotes da incubadora serão migrados para o ropensci-archive até
-  por padrão, após um ano, depois que você [orientação de transferência](#archivalguidance).
+  revisão. Os pacotes de incubadora serão migrados para o "ropensci-archive"
+  por padrão, seguindo as [orientações de transferência](#archivalguidance).
 
-- Os pacotes de incubadoras arquivados ganharão o tipo "arquivado".
+- Os pacotes de incubadoras arquivados ganharão o tipo "archived".
```

Comment:
```suggestion
- Os pacotes de incubadora arquivados ganharão o tipo "archived".
```

Reply:
The [original page](https://devguide.ropensci.org/maintenance_curation.html#incubator-packages) has some occurrences of "incubator packages" and this translation is mostly using "packages de incubadora", with the exception of this specific line.  Why the difference?

Reply:
Oh I see what you mean! I will be off these next days travelling. But once I comeback, I will look at this again. Thank you!

Reply:
That is my fault! Sorry, I missed this.  
We still haven't merged it with the book, so if you let me know the decision, you can still make the change you agree to here before the chapter becomes part of the guide.